components/datetime/Cargo.toml

```diff
@@ -67,7 +68,7 @@ bench = false  # This option is required for Benchmark CI
 std = ["icu_provider/std", "icu_locid/std", "icu_calendar/std"]
 default = ["provider_serde"]
 bench = []
-provider_serde = ["serde", "litemap/serde_serialize", "smallvec/serde", "litemap/serde", "zerovec/serde", "tinystr/serde"]
+provider_serde = ["serde", "litemap/serde_serialize", "smallvec/serde", "litemap/serde", "zerovec/serde", "tinystr/serde", "serde_with"]
```

Comment:
@zbraniecki As the dependencies tzar, are you okay adding serde_with?

Reply:
Ugh `serde_with` is not `no_std`. Back to the drawing board.

Reply:
I think we can implement something generic and stick it in `provider/core` for now (we can move it around later)